download_action.go

```diff
@@ -60,17 +60,43 @@ func DownloadHttpUrl(url, filename string, forcedRewrite bool) error {
 	return nil
 }
 
+// Validate if supported parameters passed in recipe
```

Comment:
nitpick the go convention is that the first line starts with the name of what it describes, so validateUrl in this case ;)

Reply:
done